setup.py

```diff
@@ -39,6 +39,7 @@ def remove_comments_and_empty_lines(lines):
 extra_files = [
     "pypi_package_index.json",
     "pyspark/ml/log_model_allowlist.txt",
+    "../mlflow/server/auth/basic_auth.ini",
```

Comment:
Do we need `../mlflow`?

Reply:
Not needed, but I saw that's the format being used in alembic_files a few lines above

Reply:
Got it. Let's remove it.

Reply:
Filed https://github.com/mlflow/mlflow/pull/8504